app/controllers/entities_controller.rb

```diff
@@ -1,14 +1,27 @@
 # frozen_string_literal: true
 class EntitiesController < ApplicationController
+  include Europeana::Entities
+  include Europeana::EntitiesApiConsumer
+
   def suggest
-    render json: Europeana::API.entity.suggest(entity_api_params(text: params[:text]))
+    render json: Europeana::API.entity.suggest(entities_api_suggest_params(params[:text]))
   end
 
-  protected
+  def show
+    @entity = Europeana::API.
+              entity.fetch(entities_api_fetch_params(params[:type], params[:namespace], params[:identifier])).
+              merge(__params__: { type: params[:type], namespace: params[:namespace], identifier: params[:identifier] })
```

Comment:
The only place these __params__ are being used is to generate the items_about path, which isn't ready yet and once it will be will probably be more akin to how the items_by is generated. So let's get rid of this and remove the references from the content method in entities/show.rb